app/_src/gateway/plugin-development/quickstart.md

```diff
@@ -0,0 +1,478 @@
+---
+title: Quick Start
+book: plugin_dev
+chapter: 2
+---
+
+The following instructions are written to help you quickly build, test, and run your first 
+{{site.base_gateway}} custom plugin with [Lua](http://www.lua.org/). The remaining pages in 
+this section will provide details on other advanced topics related to 
+plugin development and best practices.
+
+{:.note}
+> **Note:** The following prerequisites are required to complete this guide
+>   * A `docker` CLI and associated Docker engine
+>   * `git` 
+>   * `curl`
+
+1. **Initialize a new plugin repository**
+
+    Start by opening a terminal and changing directories to a location where you store source code. 
+    
+    Create a new folder for the plugin and navigate into it:
+
+    ```sh
+    mkdir -p my-plugin && \
+      cd my-plugin
+    ```
+
+    Next, create the plugin folder structure:
+
+    {:.note}
+    > **Important:** The specific tree structure and filenames shown in this guide are important to ensuring 
+    > the development and execution of your plugin works properly with {{site.base_gateway}}. Do not
+    > deviate from these names for this guide.
+
+    ```sh
+    mkdir -p kong/plugins/my-plugin && \
+      mkdir -p spec/my-plugin
+    ```
+
+1. **Initialize plugin modules**
+
+    Plugins are made up of [Lua modules](http://www.lua.org/manual/5.1/manual.html#5.3) defined in 
+    individual files. 
+
+    Start by creating empty `handler.lua` and `schema.lua` files, which are the minimum required modules
+    for a functioning plugin:
+
+    ```sh
+    touch kong/plugins/my-plugin/handler.lua
+    touch kong/plugins/my-plugin/schema.lua
+    ```
+  
+    The `handler.lua` module contains the core logic of your new plugin.
```

Comment:
I think I might put the file additions into substeps just to break this step up a bit.

Reply:
@cloudjumpercat can you clarify a little bit for me please?  My thinking was that, with the `touch` commands as a specific step we help clue the user in on the actual files they are going to be working with and their names and locations.  I hope that it helps the user see ahead of time what files they are about to work on and where they live.

Reply:
Maybe this is what you're thinking, but I'm going to move the `touch` commands up into the previous step, then i'm going to rename this section as "Author the Schema file" and the next section as "Author the Handler file"

Reply:
@cloudjumpercat can we resolve this comment?